client/src/redux/actions/userAction.js

```diff
@@ -20,10 +20,10 @@ export const userRegister = (username, email, password) => async (dispatch) => {
     const config = {
       "Content-Type": "application/json",
     };
-    const baseUrl="https://mydiary-backend-host.herokuapp.com";
+    const baseUrl = "https://mydiary-backend-host.herokuapp.com";
     const { data } = await axios.post(
-          baseUrl+"/users/signup",
-            { email, password, username },
+      baseUrl + "/users/signup",
```

Comment:
create a const vaiable which will be used in that const.js


```
const baseUrl = "https://mydiary-backend-host.herokuapp.com"

const apiRoutes = {
    register = '/users/signup'
}

```

and here :

`axios.post(${baseUrl + apiRoute.register})`